packages/smooth_app/lib/pages/scan/ml_kit_scan_page.dart

```diff
@@ -43,7 +43,7 @@ class MLKitScannerPageState extends State<_MLKitScannerPageContent> {
   /// we considered, that the camera should be reinitialized
   ///
   /// On a 60Hz display, one frame =~ 16 ms => 100 ms =~ 6 frames.
-  static const int _postFrameCallBackMinDelay = 100; // in milliseconds
+  static const int postFrameCallBackMinDelay = 100; // in milliseconds
```

Comment:
Suggestion: `postFrameCallbackStandardDelay` to avoid confusion with the min delay function. Or rename the function.